ui/uihandler.go

```diff
@@ -158,6 +158,7 @@ func Handler(w http.ResponseWriter, r *http.Request) {
 	grpcPingDelay, _ := time.ParseDuration(r.FormValue("grpc-ping-delay"))
 	stdClient := (r.FormValue("stdclient") == "on")
 	httpsInsecure := (r.FormValue("https-insecure") == "on")
+	resolvedIP := r.FormValue("resolvedIP")
```

Comment:
should be « resolve »

Reply:
@ldemailly Thank you so much for being patient with me, I am relative new to go, and feel honored to work gurus like you. 

I guess you will smash all the commit before merge.  Otherwise, I will do a rebase.